doc/nrf/installation/install_ncs.rst

```diff
@@ -124,7 +124,7 @@ Depending on your preferred development environment, complete the following step
       .. parsed-literal::
          :class: highlight
 
-         nrfutil toolchain-manager config --set toolchain-index=*custom_toolchain_URL*
+         nrfutil toolchain-manager config --set toolchain-index=\*custom_toolchain_URL*
```

Comment:
This does not do the trick yet. I believe this needs an empty space between the backslash and the asterisk, possibly the same combination after the second asterisk.